drivers/adc/ad7768/ad7768.h

```diff
@@ -76,38 +76,51 @@
 #define AD7768_REG_CH_GAIN_1(ch)			(0x36 + (ch) * 3)
 #define AD7768_REG_CH_GAIN_2(ch)			(0x37 + (ch) * 3)
 #define AD7768_REG_CH_GAIN_3(ch)			(0x38 + (ch) * 3)
-#define AD7768_REG_CH_SYNC_OFFSET(ch)		(0x4E + (ch) * 3)
+#define AD7768_REG_CH_SYNC_OFFSET(ch)			(0x4E + (ch) * 3)
 #define AD7768_REG_DIAG_METER_RX			0x56
 #define AD7768_REG_DIAG_CTRL				0x57
-#define AD7768_REG_DIAG_MOD_DELAY_CTRL		0x58
+#define AD7768_REG_DIAG_MOD_DELAY_CTRL			0x58
 #define AD7768_REG_DIAG_CHOP_CTRL			0x59
 
 /* AD7768_REG_CH_STANDBY */
 #define AD7768_CH_STANDBY(x)				(1 << (x))
 
 /* AD7768_REG_CH_MODE_x */
 #define AD7768_CH_MODE_FILTER_TYPE			(1 << 3)
+#define AD7768_CH_MODE_DEC_RATE_MSK			NO_OS_GENMASK(2, 0)
 #define AD7768_CH_MODE_DEC_RATE(x)			(((x) & 0x7) << 0)
 
 /* AD7768_REG_CH_MODE_SEL */
-#define AD7768_CH_MODE(x)					(1 << (x))
+#define AD7768_CH_MODE(x)				(1 << (x))
 
 /* AD7768_REG_PWR_MODE */
+#define AD7768_PWR_MODE_POWER_MODE_MSK			NO_OS_GENMASK(5, 4)
 #define AD7768_PWR_MODE_SLEEP_MODE			(1 << 7)
-#define AD7768_PWR_MODE_POWER_MODE(x)		(((x) & 0x3) << 4)
+#define AD7768_PWR_MODE_POWER_MODE(x)			(((x) & 0x3) << 4)
```

Comment:
i wouldn't mix up non-functional changes with functional ones.

Reply:
Created a separate commit for these changes.